### ISSUE TITLE
Unknown address family 0

### ISSUE THREAD

**Issue by billionaire** [nmap/nmap#283]
When i run such command:
```
nmap -p 1-65535 -T4 -A -v 185.87.49.223
```
In the process, after the start of "*Initiating NSE at HH:MM*" I always get error "*Unknown address family 0*" and nmap shuts down.
When scanning other ip-addresses or domain, the process ends correctly.

Nmap version 7.01 (downloaded from https://nmap.org/dist/nmap-7.01-setup.exe ). Version 6.49 behaved in same way.

![20160124002547lmzquwezmy](https://cloud.githubusercontent.com/assets/6255854/12533706/dba66794-c241-11e5-96b9-30a5787a1e87.png)

**Comment by gvanem** [comment]
I'm on Win-10 with the same **7.01 SVN** version and it  works here.Try adding your IP *source-address* and see if that make a difference:
```
nmap -S a.b.c.d -p 1-65535 -T4 -A -v 185.87.49.223
```

**Comment by dmiller-nmap** [comment]
@billionaire I can't see what would be causing this. Can you reduce the crash to a smaller number of scan options? For example, try `nmap -T4 -sC -d --script-trace 185.87.49.223` to eliminate the version scan, OS detection, and traceroute, while adding some diagnostic output that could narrow in on one script in particular causing the problems.

**Comment by timvanderweijde** [comment]
Hi,

I have the same error (?), the script stops after this warning. I do have another query running though.

nmap -T4 -O -A -v www.bluecape.nl

....
Completed SYN Stealth Scan at 06:05, 5.80s elapsed (1000 total ports)
Initiating Service scan at 06:05
Scanning 9 services on www.bluecape.nl (87.250.152.165)
Completed Service scan at 06:05, 12.17s elapsed (9 services on 1 host)
Initiating OS detection (try #1) against www.bluecape.nl (87.250.152.165)
Initiating Traceroute at 06:05
Completed Traceroute at 06:05, 3.07s elapsed
Initiating Parallel DNS resolution of 6 hosts. at 06:05
Completed Parallel DNS resolution of 6 hosts. at 06:05, 0.00s elapsed
NSE: Script scanning 87.250.152.165.
Initiating NSE at 06:05
Unknown address family 0

**Comment by dmiller-nmap** [comment]
@timvanderweijde Thanks for bumping this issue. Found a fix, but also found another bug that affects scanning that target. See #378 for why SSL stuff won't work right on POP3, IMAP, etc. there.

**Comment by dmiller-nmap** [comment]
Fixed in f38b959

**Comment by timvanderweijde** [comment]
@dmiller-nmap You're welcome. :-) And nice to see the fix already.
